packages/app/src/components/SearchResultsCard/index.js

```diff
@@ -1,30 +1,50 @@
 import React, { useContext, useEffect, useState } from 'react';
-import { Link } from 'react-router-dom';
+import { Redirect, Link } from 'react-router-dom';
 import { SearchResultDataContext } from '../../utils/context/SearchResultsContext';
 import './style.css';
 
 const SearchResultsCard = () => {
   const [SearchResultData, setSearchResultData] = useContext(SearchResultDataContext);
+
+
   return (
     <>
       <section className="search-result-card">
+
         <table className="search-result-table">
+
+
           <thead>
-            <tr>
-              <th>Search Results</th>
+
+
+            <tr className="search-result-exit">
+
+              <th>Search Results
+
+              </th>
+              <Link to="/">
+                <button> X </button>
+              </Link>
+
             </tr>
+
           </thead>
+
+
           <tbody>
+
             <tr>
               <td><h1><strong>Application Name</strong></h1></td>
               <td><h1><strong>Provider's Name</strong></h1></td>
               <td><h1><strong>Patient Name</strong></h1></td>
               <td><h1><strong>Received Date</strong></h1></td>
+
+
```

Comment:
Nit: I'd probably delete all of the extra blank lines that have been added, except maybe one on line 33 (between the thead and tbody).